.travis/setup-deploy.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+#Pull request is a number or false
+if [ "$TRAVIS_BRANCH" != 'master' ] || [ "$TRAVIS_PULL_REQUEST" != 'false' ]; then
+    echo "Skipping env deployment setup for a non-release build"
+    exit 0
+fi
+
+echo "Verifying environment variables"
+
+SIGNING_VARS='SONATYPE_USERNAME SONATYPE_PASSWORD GPG_EXECUTABLE GPG_KEYNAME GPG_PASSPHRASE GPG_SECRETKEY GPG_OWNERTRUST'
+for var in ${SIGNING_VARS[@]}
+do
+    if [ -z ${!var} ] ; then
+        echo "Variable $var is not set cannot setup gpg signatures"
+        exit 1
+    fi
+done
+
+echo "Setting up env for deployment"
+openssl aes-256-cbc -K $encrypted_601c881f6a91_key -iv $encrypted_601c881f6a91_iv -in .travis/codesigning.asc.enc -out .travis/codesigning.asc -d
```

Comment:
Referencing some missing files.

Reply:
Those will be added via travis CLI so it is fine for now